model/example.js

```diff
@@ -73,20 +74,70 @@ module.exports = {
         return db.selectAll(client, "select * from example_utterances");
     },
 
+    getCommandsForUser(client, language, userId, start, end) {
```

Comment:
maybe add a line to comment why we need this? it took me a while to realize...

Reply:
Sure